readme.md

```diff
@@ -121,43 +121,43 @@ All page-related content should be located inside `#content`. It could be styles
 ## API
 
 ### `pill()`
-```
+```js
```

Comment:
Remove all `js` code block languages. It's my own pseudocode for type description and not a JS at all.

Reply:
Got it.